moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -416,7 +416,13 @@ bool IKFastKinematicsPlugin::initialize(const moveit::core::RobotModel& robot_mo
   // base_frame_ and IKFAST_TIP_FRAME_ the tip_frame_ and transform to the input pose accordingly
   if (!computeRelativeTransform(tip_frames_[0], IKFAST_TIP_FRAME_, group_tip_to_chain_tip_, tip_transform_required_) ||
       !computeRelativeTransform(IKFAST_BASE_FRAME_, base_frame_, chain_base_to_group_base_, base_transform_required_))
+  {
```

Comment:
This kind of duplicates the error output generated in `computeRelativeTransform`:
https://github.com/ros-planning/moveit/blob/6395358ce41bb6d61f9f29bc0b03054bd1dbb95a/moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp#L382-L391

Reply:
but it relies therefore on computeRelativeTransform generating the output. I think this way it would be clearer why the init fails.